airflow/www/views.py

```diff
@@ -2373,7 +2373,7 @@ def _clear_dag_tis(
 
     @expose("/clear", methods=["POST"])
     @auth.has_access_dag("PUT", DagAccessEntity.TASK_INSTANCE)
-    @action_logging
+    @action_logging(event="marked cleared")
```

Comment:
How about just "cleared"?

Reply:
same question, why are you using « marked » in all the action?

Reply:
it's ok with me. but what do you do with `success`?  `successed`? :) 
maybe we have to do `marked` with all of them to be consistent?

Reply:
if the concern is width... there will be other events that have more verbose names than even these.....

Reply:
ok yeah marked cleared isn't even english.. ok it's a different thing than marking state... never mind!  i'll update i

Reply:
> same question, why are you using « marked » in all the action?

because with the state there is no way to tell that it is a user action.

there are machine-generated events recorded for `success` and `running`.

we must put something in the event name to distinguish from those and to make clear it's a user action.

it's necessariy for `marked failed` and `marked success` because someone is manually marking those states.  it's not for `cleared` because that verb implies already that it is a manual action i think.

Reply:
I think `manually cleared` would be appropriate here

Reply:
-1 on `manually`. If we remove `root` and `airflow` owners and only show actual users then it should be obvious

Reply:
so how does it look to everyone now?

i have it as 
```
cleared
marked failed
marked success
```
this seems good enough to me

Reply:
Sure. those work for me